packages/firebase_storage/lib/firebase_storage.dart

```diff
@@ -48,6 +48,13 @@ class StorageReference {
     );
   }
 
+  Future<String> getDownloadURL() async {
+    final dynamic downloadUrl = await FirebaseStorage._channel.invokeMethod(
```

Comment:
I think you can make this non-async and say `return FirebaseStorage._channel.invokeMethod(...`

See `delete` for another example of this

Reply:
Done,

Is there any advantage to having the return type here by `Future<String>` instead of `Future<dynamic>`

Reply:
It's documenting that we're returning a `String` here and not some other type like a `Uri`.